build.py

```diff
@@ -52,7 +52,7 @@ def copy_artifacts():
         execute("cp ./_spidermonkey_install/lib/libmozjs* ./python/pythonmonkey/", cwd=TOP_DIR)
 
 def build():
-    execute("git submodule update --init --recursive", cwd=TOP_DIR)
+    execute("git submodule update --init --recursive || echo 'Skipping. We are installing from an sdist tarball.'", cwd=TOP_DIR)
```

Comment:
What is the reason for the `git submodule update --init --recursive` again? If it's not necessary in build, I'm not sure why it should go in `build.py`, if this is for the tests or something like this, wouldn't it be better suited in whichever startup script is responsible for the tests.

Reply:
It's only used for the commonjs test suite right now. 

However, `setup.sh` script is also executed if install from `.tar.gz` sdist. 
https://github.com/Distributive-Network/PythonMonkey/blob/5b9881211ebd6d2747649e40ee2fcdc86564bf34/build.py#L36

Reply:
I will approve, but I think we should consider moving the submodule update to somewhere else. 

I think it's important to keep in mind the semantics of tools. In this case `build.py` is meant to be the necessary components to build the wheel. Calling `bash ./setup.sh` is absolutely necessary for building the wheel as we necessarily need the spidermonkey lib. However the commonjs test suite is not necessary for building the wheel.

Ex: If it's only needed for the commonjs test suite, perhaps peter-junior can be updated to run a setup script that runs the git submodule update.

Reply:
Agree. How about moving it to `tests/js/commonjs-modules.bash`?